src/queries/queries.js

```diff
@@ -176,31 +174,22 @@ export const buildOneYearOldTimeseriesWithAggregationByAccountId =
     }
   }
 
-export const buildOneYearBikeCommuteTimeseriesQueryByDateAndAccountId = (
-  { date, accountId },
+export const buildBikeCommuteTimeseriesQueryByAccountId = (
+  { accountId },
   enabled
 ) => {
-  const startYearDate = startOfYear(date)
-  const endYearDate = endOfYear(date)
-
   return {
     definition: Q('io.cozy.timeseries.geojson')
       .where({
         'cozyMetadata.sourceAccount': accountId,
-        startDate: {
-          $gte: startYearDate.toISOString(),
-          $lte: endYearDate.toISOString()
-        },
+        startDate: { $gt: null },
```

Comment:
You shouldn't normally need to do it, but because of a [pouchdb issue](https://github.com/pouchdb/pouchdb/issues/8385#issuecomment-1098860521), it is safer to explicitly use all the indexed attributes in the `.where` clause. Hence, this `$gt: null` trick, that simply means "take all the docs with a `startDate`"